.github/workflows/release.yml

```diff
@@ -0,0 +1,79 @@
+name: Release
+
+on:
+  push:
+    branches:
+    - master
+
+jobs:
+  build:
+    runs-on: macos-latest
+
+    steps:
+    - name: Checkout obs-studio
+      uses: actions/checkout@v2
+      with:
+        repository: obsproject/obs-studio
+        path: obs-studio
+        submodules: recursive
+        ref: 25.0.7
+
+    - name: Checkout obs-mac-virtualcam
+      uses: actions/checkout@v2
+      with:
+        path: obs-mac-virtualcam
+
+    - name: Install OBS deps
+      run: |
+        brew install FFmpeg x264 Qt5 cmake mbedtls swig
+
+    - name: Build OBS
+      run: |
+        mkdir obs-studio/build
+        cd obs-studio/build
+
+        export QTDIR=/usr/local/opt/qt 
+        cmake ..
+        make -j
+
+    - name: Build the plugin
+      run: |
+        mkdir obs-mac-virtualcam/build
+        cd obs-mac-virtualcam/build
+
+        cmake -DLIBOBS_INCLUDE_DIR:STRING=$GITHUB_WORKSPACE/obs-studio/libobs -DLIBOBS_LIB:STRING=$GITHUB_WORKSPACE/obs-studio/build/libobs/libobs.dylib -DOBS_FRONTEND_LIB:STRING=$GITHUB_WORKSPACE/obs-studio/build/UI/obs-frontend-api/libobs-frontend-api.dylib -DQTDIR:STRING=/usr/local/opt/qt ..
+        make -j
+
+    - name: Fix runtime QT deps
+      run: |
+        install_name_tool \
```

Comment:
Does this point the plugin to the QT bundled in the `.app` (instead of the system QT?)

Reply:
Yep, it's the same trick that [obs-websocket](https://github.com/Palakis/obs-websocket/blob/4.x-current/CI/package-macos.sh#L24-L31) (and OBS itself) uses to fix the libs

Reply:
Neat